### DISCUSSION TITLE
Two drawers from left and right at the same page?

### DISCUSSION THREAD

**Discussion by NikarashiHatsu** [saadeghi/daisyui#2163] [Q&A] [answered]
Is there's any reference on how can I implement two drawers at one page?

**Comment by saadeghi** [comment, marked as answer]
You can put the second drawer inside the first drawer's `drawer-content`:  
https://play.tailwindcss.com/7ImkM49Lky

But I suggest to only use 1 drawer and change the content dynamically using JS

**Reply by NikarashiHatsu** [reply]
Thanks a lot, I need both drawer in the left and right.
